src/index.js

```diff
@@ -1,5 +1,10 @@
 import isPromise from './isPromise.js';
 
+/**
+ * Note to contributors: Please also remember to check and make sure that `index.d.ts` is also up to date with
```

Comment:
This line is breaking the build: just need to break it so the character count isn't over 100. 

Build error: https://travis-ci.org/pburtchaell/redux-promise-middleware/jobs/367749777#L493